torch/ao/quantization/fx/prepare.py

```diff
@@ -1421,6 +1426,6 @@ def prepare(
         # converting List[int] to Tensor since module attribute is
         # Union[Tensor, Module]
         model._standalone_module_input_quantized_idxs = \
-            torch.tensor(input_quantized_idxs)
-        model._standalone_module_output_quantized_idxs = torch.tensor(output_quantized_idxs)
+            torch.tensor(list(input_quantized_idxs.keys()))
+        model._standalone_module_output_quantized_idxs = torch.tensor(list(output_quantized_idxs.keys()))
```

Comment:
why are we using keys here? should we use dict here as well?

Reply:
following the original idx, we should use 'keys' here.

Reply:
I feel we should keep it as dict and change the code that is handling this

Reply:
we can do this after my PR is landed: https://github.com/pytorch/pytorch/pull/73863, currently it is expecting a list of indicies I think.

Reply:
we can do this after my PR is landed: https://github.com/pytorch/pytorch/pull/73863, currently it is expecting a list of indicies I think.

Reply:
sounds good. I'll keep 'keys' in this PR.